tokio-compat/Cargo.toml

```diff
@@ -0,0 +1,50 @@
+[package]
+name = "tokio-compat"
+# When releasing to crates.io:
+# - Remove path dependencies
+# - Update html_root_url.
+# - Update doc url
+#   - Cargo.toml
+#   - README.md
+# - Update CHANGELOG.md.
+# - Create "v0.1.x" git tag.
+version = "0.1.0-alpha.1"
+edition = "2018"
+authors = ["Tokio Contributors <team@tokio.rs>"]
+license = "MIT"
+readme = "README.md"
+documentation = "https://docs.rs/tokio-compat/0.1.0-alpha.1/tokio-compat/"
+repository = "https://github.com/tokio-rs/tokio"
+homepage = "https://tokio.rs"
+description = """
+Compatibility between `tokio` 0.2 and legacy versions.
+"""
+categories = ["asynchronous", "network-programming"]
+keywords = ["io", "async", "non-blocking", "futures"]
+
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[features]
+default = ["runtime", "sink"]
+runtime = ["old-timer", "old-reactor", "old-executor", "tokio-executor", "tokio-net", "tokio-timer", "tokio-sync", "num_cpus", "tracing-core"]
+sink = ["futures-util/sink"]
+
+[dependencies]
+futures-01 = { package = "futures", version = "0.1" }
+futures-03-core = { package = "futures-core-preview", version = "0.3.0-alpha.19" }
+futures-util = { package = "futures-util-preview", version = "0.3.0-alpha.19", default-features = false, features = ["compat"] }
+
+# runtime-only
+old-timer = { package = "tokio-timer", version = "0.2", optional = true}
```

Comment:
can we go with the `reactor-01` style as used above, I generally perfer `reactor01` tbh.

Reply:
changed in 1e967d7b233299c72e1733cf4e2c1ff6faeac1e7!